src/tests/components/RunButton.test.js

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import { render } from '@testing-library/react';
 import { unmountComponentAtNode } from 'react-dom';
-import RunButton from '../components/RunButton';
+import RunButton from '../../components/RunButton';
 import { act } from 'react-dom/test-utils';
+import 'fake-indexeddb/auto';
```

Comment:
Do we need `fake-indexeddb` in this file? I am suspicious that we do not.

Reply:
I was also suspicious but any call to RunSUSHI won't go through in a testing environment without it.

Reply:
I'm fine with just ignoring my suspicions then!